### PR TITLE
KOGITO-1614 - Review process details timeline style including abort and retry operations

### PR DESCRIPTION
**Pull request by AjayJagan**
- connected with data index
 - changes in icon 
 - added skip and retry to the nodes

### CODE REVIEW

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx [hunk 1]
```diff
@@ -1,14 +1,66 @@
...
-const ProcessDetailsTimeline: React.FC<IOwnProps> = ({ data }) => {
+const ProcessDetailsTimeline: React.FC<IOwnProps> = ({
+  data,
+  handleRetry,
+  handleSkip
+}) => {
+  const [isKebabOpen, setIsKebabOpen] = useState(false);
+
+  // TODO: make the kebab menu for each timeline item work!
+  // const { isOpen } = this.state;
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx]: Comments can be removed

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx [hunk 2]
```diff
@@ -17,29 +69,115 @@ const ProcessDetailsTimeline: React.FC<IOwnProps> = ({ data }) => {
...
+                        {/* TODO: put the correct icon in depending on the state */}
+                        {/* <Tooltip content={'Active'}>
+                            <OnRunningIcon className="kogito-management-console--timeline-status" />{' '}
+                          </Tooltip> */}
+                        {/* <Tooltip content={'Completed'}>
+                          <CheckCircleIcon
+                            color="var(--pf-global--success-color--100)"
+                            className="kogito-management-console--timeline-status"
+                          />
+                        </Tooltip> */}
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx]: Commented out code can be removed

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx [hunk 3]
```diff
@@ -17,29 +69,115 @@ const ProcessDetailsTimeline: React.FC<IOwnProps> = ({ data }) => {
...
+                            </Moment>
+                          )}
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </SplitItem>
+                  <SplitItem>
+                    {
+                      <>
+                        {/* TODO: Make the dropdown work, with contents depending on the state */}
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetailsTimeline/ProcessDetailsTimeline.tsx]: Commented out TODO can be removed

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap [hunk 4]
```diff
@@ -11,115 +11,293 @@ exports[`Process Details Timeline component Sample test case 1`] = `
...
+            }
+            id=""
+            isAppLauncher={false}
+            isContentLeftAligned={false}
+            isVisible={false}
+            maxWidth="18.75rem"
+            position="top"
+            tippyProps={Object {}}
+            trigger="mouseenter focus"
+            zIndex={9999}
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap:53]: All the props that aren't changed from the default (or are required) can be removed.

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap [hunk 5]
```diff
@@ -11,115 +11,293 @@ exports[`Process Details Timeline component Sample test case 1`] = `
...
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap:241]: Props that aren't required or changing the default can be removed

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap (hunk 4 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap:53]: @srambach - This is in the test snapshot file

File: packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap (hunk 5 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Organisms/ProcessDetailsTimeline/tests/__snapshots__/ProcessDetailsTimeline.test.tsx.snap:241]: @srambach - this is also in the test snapshot file

**Pull request closed without merging**